### ISSUE TITLE
[Assistive tech and device support] Alert and status messages aren't announced without receiving focus. (11.22.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#27414]
#### VFS team name
Caregivers

#### VFS product name
Improve File Upload Error Messaging

#### Point of Contact/Reviewers
Josh Kim

---

### Platform Issue
Alert and status messages aren't announced without receiving focus.

### Issue Details
The error message is being cut off before it is announced, possibly by a loss of focus on the page. Users may not know that an error exists as a result.

### Link, screenshot or steps to recreate

https://user-images.githubusercontent.com/14154792/125811155-eaf945b7-40fb-4e4e-b0d8-81adb7d4b650.mov

### Supporting Guidance
[11.22](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 4.1.3_AA

---

### Platform Recommendation
A solution may be found in managing focus and delaying the error alert's announcement. We recommend working with your a11y specialist to better debug and find a solution to this issue.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by longmd** [comment]
@shiragoodman , in review of this, I think this may fall under the similar Design System team need as #27411. As they are working on a new file upload component, this may be something they need to be sure to incorporate as well.

**Comment by k80bowman** [comment]
@shiragoodman we'll look into it and see if it's an issue on our end.

**Comment by SarahKay8** [comment]
Focus is being sent back to "Upload a New File" `button` after uploading large or incorrect file. 

https://images.zenhubusercontent.com/133843125/050840f6-7838-4610-b8d6-402b2a852875/20221114170306234.mp4.

Error message is being read aloud via VoiceOver upon "Upload a New File" focus return.
 Content suggestion - I'd reword error message " The File extension doesn't match the file format" is a bit weird grammatically. I'd go for "The File extension isn't of the required extension format (pdf, jpg,png)

**Comment by SarahKay8** [comment]
@caw310 Actually we can leave as is. I got a second opinion. We can close this one out
